src/oc/web/actions.cljs

```diff
@@ -924,10 +964,30 @@
             reactions-data (:reactions entry-data)
             reaction-idx (utils/index-of reactions-data #(= (:reaction %) (name reaction)))
             updated-entry-data (-> entry-data
-                                (assoc :reactions-loading next-reactions-loading)
-                                (assoc-in [:reactions reaction-idx] next-reaction-data))]
+                                   (assoc :reactions-loading next-reactions-loading)
+                                   (assoc-in [:reactions reaction-idx] next-reaction-data))]
         (assoc-in db entry-key updated-entry-data)))))
 
+(defn- handle-reaction-to-comment-finish
+  "Update comment data with the reaction data from the API."
+  [db]
+  (let [org-slug (router/current-org-slug)
+        board-slug (router/current-board-slug)
+        activity-uuid (router/current-activity-id)
+        board-key (dispatcher/board-data-key org-slug board-slug)
```

Comment:
Shouldn't this use `current-board-key` instead? If not why?

Reply:
This part doesn't use `current-board-key` because we already have the board-slug. `current-board-key` has a test to see if it the user is reacting from all posts or a single board.

Reply:
right!